src/main/java/uk/gov/ons/census/casesvc/type/RefusalType.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.ons.census.casesvc.type;
+
+public enum RefusalType {
+  SOFT_REFUSAL,
```

Comment:
This file could go into the `entity` directory where the rest of the enum classes are. There's already a EventType class which is similar to this.

Reply:
created it in there originally @ryangrundy7 but moved to type package so as not specific to domain objects